flowauth/frontend/cypress/integration/add_new_user.js

```diff
@@ -0,0 +1,66 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+describe("Login screen", function() {
+  Cypress.Cookies.debug(true);
+
+  beforeEach(function() {
+    // Log in and navigate to user details screen
+    cy.login_admin();
+    cy.visit("/");
+    cy.get("#user_list").click();
+  });
+
+  it("Add Username with space", function() {
+    cy.get("#new").click();
+    // adding username with sapce
+    cy.get("#username").type("USER ");
+    //checking validation text
+    cy.get("#username-helper-text").should(
+      "have.text",
+      "You can use letters only."
+    );
+  });
```

Comment:
Can we also check that the error message vanishes when the error condition is cleared?

Reply:
Ok, I know this sounds pernickety, but we actually want to check that there's _no_ text here rather than for the absence of this _specific_ text. I'd also like to check that for _all_ the hint texts.